src/main/java/com/minecolonies/coremod/client/gui/WindowHutFarmer.java

```diff
@@ -169,14 +169,14 @@ private void pullLevelsFromHut()
      */
     private void assignmentModeClicked(@NotNull final Button button)
     {
-        if (button.getLabel().equals(LanguageHandler.format("com.minecolonies.gui.hiring.off")))
+        if (button.getLabel().equals(LanguageHandler.format("com.minecolonies.coremod.gui.hiring.off")))
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "com.minecolonies.coremod.gui.hiring.off" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1192)